source/css/_common/scaffolding/tags/folding.styl

```diff
@@ -0,0 +1,170 @@
+// gap
+$gap = 16px // base gap
+$gap-h2 = 48px
+$gap-h3 = 32px
+$gap-h4 = 16px
+$gap-paragraph = 1rem // 区块间距
+$gap-row = .5rem // 行间距
+$gap-card = $gap
+// border radius
+$border-codeblock = 4px
+// common
+$color-card = white
+$color-text = #555
```

Comment:
If the variables you define are only used in this styl file, it is recommended to add a prefix, such as `fd-` `fold-` `folding-`, etc., to avoid conflicts in different files.

Reply:
理解错我的意思了，我是说从上面三个中选个，或者你有更好的前缀也行，但你把那三个拼起来了... :joy:

Reply:
哈哈哈哈，那我修改一下提交上去

Reply:
我已经提交上去啦